Makefile

```
@@ -131,3 +132,24 @@ verify: check format test
 
 .PHONY: verify-extended
 verify-extended: check-generate check format test test-cov test-clean
+
+.PHONY: update-skaffold-deps
+update-skaffold-deps: $(YQ)
+	@GARDENER_HACK_DIR=$(GARDENER_HACK_DIR) hack/check-skaffold-deps.sh update
```

For consistency, let's use the full path to the hack scripts.

```suggestion
	@GARDENER_HACK_DIR=$(GARDENER_HACK_DIR) $(HACK_DIR)/hack/check-skaffold-deps.sh update
```